### DISCUSSION TITLE
Looking for a friend to dive in this space

### DISCUSSION THREAD

**Discussion by Igdbase** [Cyfrin/security-and-auditing-full-course-s23#167] [General]
Hi I'm Igdbase I am just finished this wonderful course from Cyfrin and co just about to pursue a new career in have set upon myself but I want a friend or tutor who could help me or teach me more about the space so we can grow our  knowledge and achieve our goals and dre in this space

**Comment by EngrPips** [comment]
Hello @Igdbase, Glad to hear you finished the course, and I wish you the best in your search for a career partner.
